packages/core/src/2d/assembler/SlicedSpriteAssembler.ts

```diff
@@ -131,15 +140,25 @@ export class SlicedSpriteAssembler {
     }
   }
 
-  static updateColor(renderer: SpriteRenderer): void {
+  static updateColor(renderer: SpriteRenderer | UIImage, alpha: number = 1): void {
```

Comment:
Same as commented above.
we have renderer: SpriteRenderer | UIImage, which includes only two classes.
If we anticipate this pattern expanding or being reused in the future, it might still be beneficial to define a type like Renderable for consistency and maintainability, even though there are currently only two classes. However, if this combination is unique to this specific case, leaving it as-is might be more straightforward.